lib/utils.js

```diff
@@ -297,3 +285,22 @@ function parseExtendedQueryString(str) {
 function newObject() {
   return {};
 }
+
+/**
+ * Helper function (requiring partial application)
+ * for setting exports.etag or exports.wetag.
+ *
+ * @param {Boolean} weak
+ * @return {Function}
+ * @api private
+ */
+
+function setTag (weak) {
```

Comment:
I just noticed this is "setTag" instead of like "setETag". Is the missing "e" a typo?